back/spec/services/xlsx_export/attendees_generator_spec.rb

```diff
@@ -0,0 +1,40 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+describe XlsxExport::AttendeesGenerator do
+  let(:service) { described_class.new }
+
+  describe 'generate_attendees_xlsx' do
```

Comment:
I think it can be valuable to have a test for the column names in different languages. (For example, I recently wrote a similar kind of spec here https://github.com/CitizenLabDotCo/citizenlab/pull/7876/files#r1601963097). But if you do, I would recommend adding the translations to the spec fixtures translation files as well, and matching on exact strings (instead of calling I18n.t inside the spec).